zowe-api-dev/src/commands/zosbuild.ts

```diff
@@ -42,21 +41,10 @@ export default class ZosBuild extends Command {
             execSshCommandWithDefaultEnv(projectConfig.buildCommand, zosDir, env);
             for (const [zosFile, targetFile] of Object.entries(projectConfig.buildFiles)) {
                 this.log(`Downloading ${zosDir}/${zosFile} to ${targetFile}`);
-                if (f.flags.curl) {
```

Comment:
Why curl was removed?

Reply:
It was only a temporary workaround for a problem on River.